pkgs/desktops/plasma-5/default.nix

```diff
@@ -123,12 +123,13 @@ let
       kscreen = callPackage ./kscreen.nix {};
       kscreenlocker = callPackage ./kscreenlocker.nix {};
       ksshaskpass = callPackage ./ksshaskpass.nix {};
-      ksysguard = callPackage ./ksysguard.nix {};
```

Comment:
Should we add a throw alias for ksysguard?

Reply:
That would be a reasonable thing to do, yes. It should point people to plasma-systemmonitor.

Reply:
`ksysguard` still exists, it's just outside of plasma default distribution, but ideally we should keep it as an optional package on kde-apps for anyone interested.

Reply:
Best i can do - add throw alias, ksysguard just doesn't build with latest libksysguard, and i am too lazy to package old versions of all of its dependencies :D